runhouse/rns/rns_client.py

```diff
@@ -33,6 +39,7 @@ class ResourceStatusData(BaseModel):
     env_servlet_processes: Dict[str, Dict[str, Any]]
     server_pid: int
     runhouse_version: str
+    send_to_den_resp: Optional[UpdateDenResponse] = None
```

Comment:
this feels a bit out of place to include for the resource status data

Reply:
yeah, basically I agree. I do wonder if there is a better place to include the response, so it will be propagated back to `cluster.status()` without making `ClusterServlet.status()` to return two values (status itself + send to den response info)

The `send_to_den_resp` is for internal use only, and won't be displayed to users as part of status (only case it will "displayed" is by being logged if necessary)

Reply:
oh i see, this seems to be a question of how necessary it is to have that propagate all the way back, I think it really only matters when `send_to_den=True`. 

But that still doesn't solve how to actually return the data from the `atatus call`. maybe we make it a dict instead? meaning `astatus` returns  a dict with the status data + send to den resp, and then within cluster `status` we can then surface the send to den call result (where relevant), but still only return the status data like we do now?